examples/offline_profile.py

```diff
@@ -74,14 +74,15 @@ def run_profile(context: ProfileContext, csv_output: Optional[str],
         sys.exit(-1)
 
     for i in range(batch_size):
-        llm.llm_engine.add_request(
-            request_id=f"seq{i}",
-            prompt=None,
-            prompt_token_ids=torch.randint(
-                128,  # 128 to skip over special tokens
-                llm.llm_engine.model_config.get_vocab_size() // 2,
-                size=(prompt_len, )).tolist(),
-            sampling_params=sampling_params)
+        prompt_token_ids = torch.randint(
+            128,  # 128 to skip over special tokens
+            llm.llm_engine.model_config.get_vocab_size() // 2,
```

Comment:
is there a standard less hacky way to do this?

Reply:
Not that I know of, but it looks like there `ignore_eos` option in `SamplingParams` which can be set so we don't have to worry about special tokens (basically we were getting premature EOSs ending the profiling and restricting the vocab to not have special tokens fixed it, but `ignore_eos` is a more robust way of solving this)